common/src/main/java/com/wynntils/mc/mixin/CapeLayerMixin.java

```diff
@@ -40,4 +47,28 @@ private void render(
             ci.cancel();
         }
     }
+
+    @ModifyArg(
+            method =
+                    "render(Lcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;ILnet/minecraft/client/player/AbstractClientPlayer;FFFFFF)V",
+            at =
+                    @At(
+                            value = "INVOKE",
+                            target =
+                                    "net/minecraft/client/renderer/MultiBufferSource.getBuffer(Lnet/minecraft/client/renderer/RenderType;)Lcom/mojang/blaze3d/vertex/VertexConsumer;"))
+    private RenderType setTranslucenceCapeRenderType(
+            RenderType original,
+            @Local(argsOnly = true) AbstractClientPlayer livingEntity,
+            @Local PlayerSkin playerSkin) {
+        Minecraft minecraft = Minecraft.getInstance();
+        boolean isBodyVisible = !livingEntity.isInvisible();
+        boolean translucent = !isBodyVisible && !livingEntity.isInvisibleTo(minecraft.player);
+
+        PlayerFeatureRenderTranslucentCheckEvent.Cape event = new PlayerFeatureRenderTranslucentCheckEvent.Cape(
+                translucent, livingEntity, translucent ? 0.15f : 1.0f, ((PlayerModelExtension)
```

Comment:
Why the hard-coded 0.15f? That is something the Feature can figure out itself as long as the `translucent` argument is passed in the event.